llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -6743,8 +6743,15 @@ static bool SwitchToLookupTable(SwitchInst *SI, IRBuilder<> &Builder,
     TableSize =
         (MaxCaseVal->getValue() - MinCaseVal->getValue()).getLimitedValue() + 1;
 
+  // If the default destination is unreachable, or if the lookup table covers
+  // all values of the conditional variable, branch directly to the lookup table
+  // BB. Otherwise, check that the condition is within the case range.
+  bool DefaultIsReachable =
+      !isa<UnreachableInst>(SI->getDefaultDest()->getFirstNonPHIOrDbg());
```

Comment:
```suggestion
  bool DefaultIsReachable = !SI->defaultDestUndefined();
```

Reply:
Changed! Didn't realize that was a thing, I just copied the existing code :P